src/projetos/SugestaoPage.tsx

```
@@ -76,25 +78,50 @@ export function SugestaoPage() {
         toast.info('Cadastrando sugestão...')
+        console.log('Valores do formulário:', values)
+
         const cursoSelecionado =
             values.curso && curso.length > 0
                 ? curso.find((c) => c.id?.toString() === values.curso)
                 : undefined
 
-        const professorSelecionado = professores.find((p) => p.email === values.professores[0])
-
-        if (!cursoSelecionado || !professorSelecionado || !alunoLogado?.id) {
+        const professoresSelecionados = professores.filter((prof) =>
```

ESLint

[error] Delete `····`

(prettier/prettier)